source/Irrlicht/CIrrDeviceSDL.h

```diff
@@ -303,6 +303,8 @@ namespace irr
 
 		bool Resizable;
 
+		core::rect<s32> *lastElemPos;
```

Comment:
Why a pointer? Just use `std::optional` here, saves you the manual memory management headache.

Reply:
Thanks for the tip. (I am more used to C than C++ and was not aware of `std::optional`.)